cmd/agentless-scanner/awsutils/config.go

```diff
@@ -0,0 +1,143 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package awsutils
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/DataDog/datadog-agent/cmd/agentless-scanner/types"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/datadog-agent/pkg/version"
+	"golang.org/x/time/rate"
+
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+	ddogstatsd "github.com/DataDog/datadog-go/v5/statsd"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/aws/middleware"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/feature/ec2/imds"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	smithymiddleware "github.com/aws/smithy-go/middleware"
+)
+
+const (
+	// DefaultSelfRegion is the default region for the self identity.
+	//
+	// TODO: we should have this as part of our configuration to be able to
+	// bootstrap ourselves in the correct region.
+	DefaultSelfRegion = "us-east-1"
+)
+
+var (
+	statsd *ddogstatsd.Client
```

Comment:
Our team is working on removing such global states because in the mid long term it creates many hard to catch bugs.

I would kindly advice to not add these kind of global states as it is hard to remove later.